src/modules/Home/components/AccountBalances/AccountTotalBalances.tsx

```diff
@@ -1,20 +1,27 @@
 import { BoxProps } from '@chakra-ui/react';
 
 import TotalBalance from '@/components/Common/TotalBalance';
-import { useGetCoingeckoCoin } from '@/hooks/queries/useGetCoingeckoCoin';
+import { usePrice } from '@/hooks/queries/usePrice';
```

Comment:
Should be useGetCSPRMarketData for more clear, with useGet as prefix for queries hook